### ISSUE TITLE
Should we Separate Physical Intrusion and Single Point of Failure?

### ISSUE THREAD

**Issue by petermetz** [hartm/SecurityTaskForce#10]
I added the infra threat according to what we discussed during the task force call the other day, but while I was typing it out, it made me wonder whether the threat should be broken down into two separate ones where 
- SPoF is a threat of an architectural design flaw and the
- physical intrusion is just a generic threat that we don't give a specific example for because if you are physically in a data center, that gives you a million different options to compromise the system, your options are virtually limitless and so narrowing that down to a single example might not be what we want.

Thoughts?

**Comment by arsulegai** [comment]
Agree, we can consider them as separate threats. Single point of failure could also arise due to bad architecture and implementation.

**Comment by hartm** [comment]
This would be a great thing to reflect in the document, not just the github issues, so feel free to add essentially this text justifying this to the document.

Thanks for spending time on this!

**Comment by petermetz** [comment]
@arsulegai @hartm Thank you for the feedback. I'll update the document based on this and what we talked about in today's meeting.
